public/documentation/embedding_crosswalk/crosswalk_aar.md

```diff
@@ -114,6 +114,8 @@ Then install it into the local Maven repository:
     }
     ```
 
+    Using mavenLocal() instead of remote repo url to refer local aar.
+
```

Comment:
we should add following scripts in addition below "Use mavenLocal() instead of the remote repo url to refer to a local aar."
Repositories {
     mavenLocal()
}